bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/BusyIndicator.java

```diff
@@ -70,6 +70,36 @@ public static void showWhile(Display display, Runnable runnable) {
 		}
 	}
 
+	/**
+	 * If called from a {@link Display} thread waits for the given
```

Comment:
Picky things.  An "if" clause without a "then" should be separated by a coma from the "then" part, so a coma after "thread".